### DISCUSSION TITLE
新增邮件服务器

### DISCUSSION THREAD

**Discussion by stevending1st** [kaiyuanshe/RFC#5] [Ideas]
目前我们采用的企业邮箱供应商要求每月只能向外部发送 500 封邮件，但是，我们未来会涉及的邮件业务：

- 开源年报动态通知
- 开源年报调查
- 内部进度同步
- ...

如果单纯发送应该够了，但是每次订阅会有 2 封邮件，如果业务线过多，怕扛不住。

建议建立邮件服务器，最好可以绑定二级域名...

**Comment by Guanngxu** [comment]
暂时研究下个人邮箱

**Comment by wangrunlin** [comment]
飞书邮箱可以使用自定义域名，并且支持公共邮箱

- [飞书邮箱管理员使用手册](https://www.feishu.cn/hc/zh-CN/articles/207987789616)
